test/crs/conversions.jl

```diff
@@ -1365,6 +1365,39 @@
       @inferred convert(LatLon, c2)
     end
 
+    @testset "LatLon <> Albers" begin
```

Comment:
In our tests, we use PROJ results to verify that our results are correct.
To generate PROJ results, you can use the Proj.jl package:
```julia
julia> using Proj

julia> trans = Proj.Transformation("EPSG:4269", "EPSG:5070") # LatLon{NAD83} to AlbersUS
Transformation pipeline
    description: Conus Albers
    definition: proj=pipeline step proj=axisswap order=2,1 step proj=unitconvert xy_in=deg xy_out=rad step proj=aea lat_0=23 lon_0=-96 lat_1=29.5 lat_2=45.5 x_0=0 y_0=0 ellps=GRS80
    direction: forward


julia> trans(45, 90)
(-7.231430540202629e6, 1.1853758709623523e7)

julia> trans(-45, 90)
(-1.5156419949174397e7, 1.3963188032694416e7)

julia> trans(45, -90)
(472145.2567221438, 2.4606302944375253e6)

julia> trans(-45, -90)
(989573.4665648951, -5.723953827495912e6)
```